conda_forge_tick/update_upstream_versions.py

```diff
@@ -152,6 +153,18 @@ def get_version(self, url):
         return str(ver).replace("-", "_")
 
 
+def get_sha256(url):
```

Comment:
Would it be worthwhile to use https://github.com/regro/cf-scripts/blob/master/conda_forge_tick/migrators.xsh#L272 here?

Reply:
I'd like to keep this file pure python if you don't mind

Reply:
You can import from xonsh files within python, `from rever.tools import hash_url` should work within python.

Reply:
Sure, but I usually import this file in ipython to debug it and it works with only a minor change. I'd like to keep my development workflow the same if it's not too much trouble.

Reply:
Ok

Reply:
I don't really understand what the workflow difference is.  You could still import this file in IPython and debug it...

Reply:
I don't have a xonsh installation and please don't force me to do so.